Turbolinks/Visit.swift

```diff
@@ -15,6 +15,8 @@ protocol VisitDelegate: AnyObject {
     func visitRequestDidStart(_ visit: Visit)
     func visit(_ visit: Visit, requestDidFailWithError error: NSError)
     func visitRequestDidFinish(_ visit: Visit)
+    
+    func visitDidTerminate(_ visit: Visit)
```

Comment:
The name `visitDidTerminate` feels a bit confusing. Maybe something like `webViewprocessDidTerminate`?

On the other hand, I do see the pattern of having a prefix of `visit` for all the functions here.

🤔

Reply:
yea, the convention with all of the other protocol methods is why i stuck with `visit`